build_requirements.txt

```
@@ -1,2 +1,2 @@
-flake8>=3.5.0,<3.9.0
-flake8-import-order>=0.18.1
+jupyterlab>=3.0.17
+jupyter-packaging>=0.10,<2
```

Could we please remove this cap?
```suggestion
jupyter-packaging>=0.10
```

---

sure, I only left that in because the migration guide had it when I did the initial changes to setup.py